client/src/featureform/serving.py

```diff
@@ -701,13 +702,17 @@ def compute_feature(self, f_name, f_variant, entity_name):
                     f_name, f_variant, Scalar(ScalarType(feature["data_type"]))
                 )
             else:
+                print("Creating index")
                 table = provider.create_index(
                     f_name,
                     f_variant,
                     VectorType(
                         ScalarType(feature["data_type"]), feature["dimension"], True
                     ),
                 )
+                print("Waiting for index to finalize")
+                time.sleep(60)
```

Comment:
should we let the user know how long the wait is? i.e. "Waiting for index to finalize: Sleeping 60 seconds"